portfolio/src/main/webapp/script.js

```diff
@@ -39,15 +39,34 @@ function addRandomQuote() {
  * Fetches data and handles response by converting to text for the homepage
  */
 function getDataHomepage() {
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementById('quote-container').innerText = quote;
+  fetch('/data').then(response => response.text()).then((messages) => {
+    document.getElementById('quote-container').innerText = messages;
+    console.log(messages);
   });
+}
 
-  fetch('/data').then(response => response.json()).then((messages) => {
-      console.log(messages);
+/**
+ * Fetches data and handles response for contact page comments
+ */
+function getDataComments() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+      // Build list of comments
+      const commentsEl = document.getElementById('comments-list');
```

Comment:
Abbreviations can sometimes be confusing. (Write for the readers...).
I think `commentsElement` or at least `commentsElem` would be clearer. Google's style guide strong encourages the former, except for very common abbreviations (like "Id" or "ID" instead of "identifier").